### PR TITLE
Editted Search Bar with more entries

### PR DESCRIPTION
**Pull request by Mmendero**
++ Added two inputs for searching content ("Type" and "Submit")
-- Changed Intro text to add website goal/target
-- Updated contacts page with placeholder information
-- Adjusted navigation bar hyperlinks

site page

### CODE REVIEW

**Comment by Mmendero** [comment]
With my latest commit, I:

- Deleted/renamed the files **input.html**, **output.html**, and **test.html** as I integrated them with the website and have allowed for navigation to these features within the site. (input.html can be accessed using the **"Add Listing"** button on navigation bar)

image


- Editted the **script.js** file's **search()** function to read the altered elements within the front page to make the searching feature on the front page functional

**New:**
image
**After Searching:**
image

File: Site_Files/src/main/webapp/add.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                <a href="./index.html">Home</a>
+                <a class="active" href="./add.html">Add Listing</a>
+                <a href="./contacts.html">Contact</a>
+                <a href="./login.html">Login/Signup</a>
+            </div>
+        </div>
+
+        <form action="/inf" method="POST">
+            <p>Enter name of your want to share:</p>
+            <textarea name="name-imput">name</textarea>
```
Comment (matthewtlam) [inline Site_Files/src/main/webapp/add.html:25]: NITPICK: input is spelt wrong for the "name" for all the textareas

**Review by matthewtlam — APPROVED** [review]
Everything looks good. It was a great idea to separate the input form as a separate page. I was also thinking about making the "type" category as a selector with options but you beat me to it. I'm not looking forward to all the merge conflicts I'm gonna have though later, but great job!

**Comment by ben188ben188** [comment]
read

**Review by ben188ben188 — CHANGES_REQUESTED** [review]
Looks good

**Review by ben188ben188 — APPROVED** [review]

File: Site_Files/src/main/webapp/index.html [hunk 2]
```diff
@@ -21,29 +21,43 @@
...
         </div>
     </div>
 
     <!--Main content of page (Options/Search Bar)--> 
-    <div class="content">
-        <p>This is the intro paragraph</p>
-        <p id = "prompt">The Opportunities i'm looking for are...</p>
-        <input type="text" placeholder="Search ...">
+    <div id="content">
+        <p id="intro">Welcome to [REDACTED]! Our goal is to expand your reach and radar when it comes to furthering your career in CS. Here, you'll be able to look for a wide variety of different resources help you learn a variety of CS-related topics from internships to textbooks!</p>
```
Comment (perrycate) [inline Site_Files/src/main/webapp/index.html:36]: Lol redacted

**Review by perrycate — APPROVED** [review]

**Comment by perrycate** [comment]
Btw thanks for including screensshots in your PR, was super helpful!

**Pull request merged**